components/eamxx/tests/single-process/mam/aci/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+include (ScreamUtils)
+
+set (TEST_BASE_NAME mam4_aci_standalone)
+set (FIXTURES_BASE_NAME ${TEST_BASE_NAME}_generate_output_nc_files)
+
+# Create the test
+CreateADUnitTest(${TEST_BASE_NAME}
+  LABELS mam4_aci physics
+  LIBS mam
+  MPI_RANKS ${TEST_RANK_START} ${TEST_RANK_END}
+  FIXTURES_SETUP_INDIVIDUAL ${FIXTURES_BASE_NAME}
+)
+
+# Set AD configurable options
+set (ATM_TIME_STEP 1800)
+SetVarDependingOnTestSize(NUM_STEPS 2 5 48)  # 1h 2.5h 24h
+set (RUN_T0 2021-10-12-45000)
+
+## Copy (and configure) yaml files needed by tests
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/input.yaml
+               ${CMAKE_CURRENT_BINARY_DIR}/input.yaml)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/output.yaml
+               ${CMAKE_CURRENT_BINARY_DIR}/output.yaml)
+
+# Ensure test input files are present in the data dir
+GetInputFile(scream/init/${EAMxx_tests_IC_FILE_MAM4xx_72lev})
+
+# Compare output files produced by npX tests, to ensure they are bfb
+include (CompareNCFiles)
+
+CompareNCFilesFamilyMpi (
+  TEST_BASE_NAME ${TEST_BASE_NAME}
+  FILE_META_NAME ${TEST_BASE_NAME}_output.INSTANT.nsteps_x1.npMPIRANKS.${RUN_T0}.nc
+  MPI_RANKS ${TEST_RANK_START} ${TEST_RANK_END}
+  LABELS mam4_aci physics
+  META_FIXTURES_REQUIRED ${FIXTURES_BASE_NAME}_npMPIRANKS_omp1
+)
+
+# Check tendency calculation
+foreach (NRANKS RANGE ${TEST_RANK_START} ${TEST_RANK_END})
```

Comment:
This looks like a copy+paste overlook. Your test does not compute any tendencies, so this test will likely fail. Let's remove it.

Reply:
Thanks for catching that. I have removed this section.